### PR TITLE
Create SanGhubFile recommendations

### PR DESCRIPTION
**Pull request by ghost**
This Pull Requests closes issue #87.

### CODE REVIEW

File: SanGhubFile,md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Maryland
+
+# Creatign a new File
```
Comment (ghost) [inline SanGhubFile,md]: Adding comment to the SanGhubfile :smile:

File: SanGhubFile,md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+## Maryland
```
Comment (swong888) [inline SanGhubFile,md:1]: @SanthoshSubramaniam Where is Maryland?

Comment (ghost) [inline SanGhubFile,md:1]: It's close to Washington DC :smiley:

**Comment by ghost** [comment]
:shipit: :+1:

**Pull request merged**